CONTRIBUTING.md

```diff
@@ -96,11 +96,25 @@ brew install postgresql
 
 ### Installation
 
-First make sure that you set up your `virtualenv` as described in [Setting up an environment](#setting-up-an-environment).  Also ensure you have the latest version of pip installed with `pip install --upgrade pip`. Next, install `dbt-core` (and its dependencies) with:
+First make sure that you set up your `virtualenv` as described in [Setting up an environment](#setting-up-an-environment).  Also ensure you have the latest version of pip installed with `pip install --upgrade pip`. Next, install `dbt-core` (and its dependencies) either with or without `pre-commit`:
 
+
+#### Installation with `pre-commit`:
 ```sh
 make dev
-# or
+```
+or, alternatively:
+```sh
+pip install -r dev-requirements.txt -r editable-requirements.txt
+pre-commit install
+```
+
+#### Installation without `pre-commit`:
+```sh
+make dev_req
+```
+or, alternatively:
+```sh
```

Comment:
Given the addition of the `dev_req` rule, adding this section makes sense on first blush.

But... the contributing file already seems _unbearably_ long to me. I wish it was like 75% shorter rather than even just a few lines longer.

Anyone that knows good reasons why they don't want the `pre-commit install` step probably has keen enough eyes to know that they can omit it and just run:
```
pip install -r dev-requirements.txt -r editable-requirements.txt
```

This is a case in which "less is more" and we can just omit explicitly documenting how to do it.
```suggestion
```

Reply:
@gshank I would prefer to exclude additional instructions to describe how to omit installation of `pre-commit` hooks for three main reasons:
1. Installing pre-commit hooks is the happy path!
1. How to omit installation of `pre-commit` seems obvious without explanation needed
1. More text makes it harder to read the document as a whole

How would you feel about going back to the more simple instructions?

Reply:
I see no need to describe how to use dev_req. Just have a Makefile target.

Reply:
Sounds good @gshank!

@justbldwn If you commit my two suggestions (including the one to remove the "Installation without `pre-commit`" section), I think we'll be good to go!

You'll probably want to make a final pass through your edits to make sure everything reads the way you want it to.

Reply:
@dbeatty10 @gshank thank you both for the helpful feedback here! i just pushed up a new commit to remove a chunk from the `CONTRIBUTING.md` about using `make dev_req`, and instead just left instructions for using `make dev`. however, the `dev_req` target in the `Makefile` still exists for anyone that may want to use it.